airflow/dags/gtfs_views/gtfs_schedule_fact_daily_feeds.sql

```diff
@@ -50,17 +50,23 @@ daily_feeds AS (
 
 feed_status AS (
     SELECT
-        daily_feeds.*
-        , CASE WHEN Status.status = "success" THEN "success" ELSE "error"
+        daily_feeds.*,
+        CASE WHEN download_status.status = "success" THEN "success" ELSE "error"
             END
-            AS extraction_status
+            AS extraction_status,
+        parse_result.parse_error_encountered
 
     FROM daily_feeds
-    LEFT JOIN `gtfs_schedule_history.calitp_status` Status
+    LEFT JOIN `gtfs_schedule_history.calitp_status` download_status
```

Comment:
Hey, can we discuss style changes that aren't necessary for features being added, before you add them? Conventions on the placement of commas, and table naming choices / camel casing are useful to set across the project, but when they're dropped into code changes without discussion, it takes longer to identify code changes made.

Reply:
I think your changes are pretty in line with DBT's style guide, so this isn't an opinion on the style changes themselves, but the rolling out of style changes..!

https://github.com/dbt-labs/corp/blob/master/dbt_style_guide.md

Reply:
I think it'd be great to follow style conventions. I don't even know there was a convention for SQL, so happy to follow whatever was being followed if that was happening. I'm also a big fan of using linters to enforce style conventions.